src/recipe/test/recipe.factory.ts

```diff
@@ -0,0 +1,41 @@
+import { faker } from '@faker-js/faker';
+
+type CreateRecipeOverrides = {
+  title?: string;
+  description?: string;
+  ingredients?: string;
+  preparation?: string;
+  isPublic?: boolean;
+};
+
+export const createRecipe = (overrides: CreateRecipeOverrides = {}) => ({
+  title: overrides.title ?? faker.word.noun(5),
+  description: overrides.description ?? faker.lorem.text(),
+  ingredients: overrides.ingredients ?? faker.lorem.word(8),
+  preparation: overrides.preparation ?? faker.lorem.lines(4),
+  isPublic: overrides.isPublic ?? true,
```

Comment:
should be `false` by default I think (to be consistent with the default `false` value in the database